module/VuFind/tests/unit-tests/src/VuFindTest/Connection/LibGuidesTest.php

```diff
@@ -66,6 +66,36 @@ public function testGetAccounts()
         $this->assertEquals('Melvil', $dewey->first_name);
     }
 
+    /**
+     * Test loading AZ.
+     *
+     * @return void
+     */
+    public function testGetAz()
```

Comment:
Might as well include typing:

```suggestion
    public function testGetAz(): void
```